packages/azure-kusto-data/src/client.ts

```diff
@@ -28,6 +28,8 @@
     QueryV1 = "queryv1",
 }
 
+export type RequestEntity = { query?: string; stream?: any; blob?: string };
```

Comment:
You can make it into a union type to be safer:

```ts
export type RequestEntity = {query: string} | {stream: any} | {blob: string};
const entity: RequestEntity;
if ("query" in entity) {
           entity.query // works
} else if ("stream" in entity) {
 entity.stream
} else {
         entity.blob // typescript knows that blob isn't null
}
```

You can't have accidentally two at the same time, and if you add another option compilation will fail until you fix the else.